compiler/runtime/Helpers.inc

```diff
@@ -64,113 +61,26 @@ SETVAL(TR_jitProfileParseBuffer,TR_jitProfileBigDecimalValue+1)
-SETVAL(TR_jitRetranslateCaller,TR_incrementCycleCounter+1)
-SETVAL(TR_jitRetranslateCallerWithPrep,TR_jitRetranslateCaller+1)
-SETVAL(TR_addPackedDecimal,TR_jitRetranslateCallerWithPrep+1)
-SETVAL(TR_subtractPackedDecimal,TR_addPackedDecimal+1)
-SETVAL(TR_multiplyPackedDecimal,TR_subtractPackedDecimal+1)
-SETVAL(TR_multiplyPackedDecimalFast,TR_multiplyPackedDecimal+1)
-SETVAL(TR_dividePackedDecimal,TR_multiplyPackedDecimalFast+1)
-SETVAL(TR_packedDecimalToFloat,TR_dividePackedDecimal+1)
-SETVAL(TR_packedDecimalToDouble,TR_packedDecimalToFloat+1)
-SETVAL(TR_packedDecimalToLongDouble,TR_packedDecimalToDouble+1)
-SETVAL(TR_floatToPackedDecimal,TR_packedDecimalToLongDouble+1)
-SETVAL(TR_doubleToPackedDecimal,TR_floatToPackedDecimal+1)
-SETVAL(TR_longDoubleToPackedDecimal,TR_doubleToPackedDecimal+1)
-SETVAL(TR_packedDecimalToDecFloat,TR_longDoubleToPackedDecimal+1)
-SETVAL(TR_packedDecimalToDecDouble,TR_packedDecimalToDecFloat+1)
-SETVAL(TR_packedDecimalToDecLongDouble,TR_packedDecimalToDecDouble+1)
-SETVAL(TR_decFloatToPackedDecimal,TR_packedDecimalToDecLongDouble+1)
-SETVAL(TR_decDoubleToPackedDecimal,TR_decFloatToPackedDecimal+1)
-SETVAL(TR_decLongDoubleToPackedDecimal,TR_decDoubleToPackedDecimal+1)
-SETVAL(TR_packedDecimalToSInt32,TR_decLongDoubleToPackedDecimal+1)
-SETVAL(TR_packedDecimalToUInt32,TR_packedDecimalToSInt32+1)
-SETVAL(TR_packedDecimalToSInt64,TR_packedDecimalToUInt32+1)
-SETVAL(TR_packedDecimalToUInt64,TR_packedDecimalToSInt64+1)
-SETVAL(TR_floatToSInt32,TR_packedDecimalToUInt64+1)
-SETVAL(TR_floatToSInt64,TR_floatToSInt32+1)
-SETVAL(TR_floatToUInt32,TR_floatToSInt64+1)
-SETVAL(TR_floatToUInt64,TR_floatToUInt32+1)
-SETVAL(TR_doubleToSInt32,TR_floatToUInt64+1)
-SETVAL(TR_doubleToSInt64,TR_doubleToSInt32+1)
-SETVAL(TR_doubleToUInt32,TR_doubleToSInt64+1)
-SETVAL(TR_doubleToUInt64,TR_doubleToUInt32+1)
-SETVAL(TR_longDoubleToSInt32,TR_doubleToUInt64+1)
-SETVAL(TR_longDoubleToSInt64,TR_longDoubleToSInt32+1)
-SETVAL(TR_longDoubleToUInt32,TR_longDoubleToSInt64+1)
-SETVAL(TR_longDoubleToUInt64,TR_longDoubleToUInt32+1)
-SETVAL(TR_floatToDecimalFloat,TR_longDoubleToUInt64+1)
-SETVAL(TR_floatToDecimalDouble,TR_floatToDecimalFloat+1)
-SETVAL(TR_floatToDecimalLongDouble,TR_floatToDecimalDouble+1)
-SETVAL(TR_DFPToBinaryFloat,TR_floatToDecimalLongDouble+1)
-SETVAL(TR_DFPToBinaryDouble,TR_DFPToBinaryFloat+1)
-SETVAL(TR_DFPToBinaryLongDouble,TR_DFPToBinaryDouble+1)
-SETVAL(TR_DFPToHexFloat,TR_DFPToBinaryLongDouble+1)
-SETVAL(TR_DFPToHexDouble,TR_DFPToHexFloat+1)
-SETVAL(TR_DFPToHexLongDouble,TR_DFPToHexDouble+1)
-SETVAL(TR_jitProfileStringValue,TR_DFPToHexLongDouble+1)
-SETVAL(TR_jitCheckIfFinalizeObject,TR_jitProfileStringValue+1)
-SETVAL(TR_comparePackedDecimal,TR_jitCheckIfFinalizeObject+1)
-SETVAL(TR_COBOLRTMsg, TR_comparePackedDecimal+1)
-SETVAL(TR_packedDecimalPower, TR_COBOLRTMsg+1)
-SETVAL(TR_initialInvokeExactThunk_unwrapper,TR_packedDecimalPower+1)
+SETVAL(TR_emilyCallGlue,TR_referenceArrayCopy+1)
```

Comment:
I believe `TR_emilyCallGlue` can go as well.  There is one reference to it in OMR, but that can be deleted.

Reply:
Yes, I think that can be deleted as well.
However, I have removed only those enums which had no presence elsewhere apart from Helpers.inc.
So, I think there might a few more similar to emilyCallGlue which have presence in other file (just in switch-case).

Reply:
Would you happen to remember which others?  I can quickly eyeball them and let you know if they can be deleted.

Reply:
TR_AbstractMethodError
TR_IllegalAccessError
PyHelper_TupleGetItem
PyHelper_BinaryRemainder
RubyHelper_vm_send_simple

And similarly other Python and Ruby helper enums.

Reply:
Leave the Py and Ruby helpers for now.  `TR_AbstractMethodError` and `TR_IllegalAccessError` can go.

Reply:
Addressed in 8d71b30